### DISCUSSION TITLE
difference in launch speed of ComfyUI in Local & Cloud Service.

### DISCUSSION THREAD

**Discussion by JaypTookMyJayp** [comfyanonymous/ComfyUI#5484] [Q&A]
in my GPU cloud service, it takes ~40 seconds to launch ComfyUI. I'm talking about after the container spins up.
in my local computer, it takes ~10 seconds to launch, also it has wayyy more custom nodes.
what makes the difference?
my cloud service has way better GPUs too!

**Comment by Lalimec** [comment]
Same here. This has been discussed before but i dont remember any solutions, lets see if someone has a an idea about this time.

**Comment by ltdrdata** [comment]
In the case of cloud services, the python dependency reinstallation step is executed every time an instance is run.

**Reply by Lalimec** [reply]
are there any resources that i can read about this, or can i do anything to improve the initialization times?

**Reply by ltdrdata** [reply]
Each cloud service has different characteristics. I am explaining based on Google Colab.

If you're using a deployed container with an image that has all dependencies pre-installed, it could be shortened. 
However, with Colab, there's no way to do this at all.

**Reply by Lalimec** [reply]
Mine were aws and lambdalabs actually. Thats why i was kinda weirded out by this difference. Especially in aws.
